### ISSUE TITLE
Chrome browser not terminated after execution - initial fix implemented

### ISSUE THREAD

**Issue by kensoh** [kelaberetiv/TagUI#648]
Issue first spotted on macOS Catalina release, running Chrome Version 79.0.3945.88 (Official Build) (64-bit). Not sure if it is due to macOS update changes or Chrome update change. 

Observations from macOS shows that the Chrome processes info returned have changed. Due to that, the TagUI Chrome process isn't killed properly. Adding another `grep window-size` after the `grep tagui_user_profile` seems to fix the issue. To dig further and also collect feedback.

Below debugging info from macOS. Need to check on Linux and Windows. CC @siowyisheng 

```
Tebel:src kensoh$ ps x | grep remote-debugging-port=9222 | grep tagui_user_profile | sed -e 's/^[ ]*//'                 
14058 s000  S      0:18.41 /Applications/Google Chrome.app/Contents/MacOS/Google Chrome --user-data-dir=chrome/tagui_user_profile --remote-debugging-port=9222 about:blank --window-size=1366,842
14361 s000  S      0:00.15 /Applications/Google Chrome.app/Contents/Frameworks/Google Chrome Framework.framework/Versions/79.0.3945.88/Helpers/Google Chrome Helper (Renderer).app/Contents/MacOS/Google Chrome Helper (Renderer) --type=renderer --remote-debugging-port=9222 --field-trial-handle=1718379636,15956115326844771816,7185594664229212159,131072 --lang=en-GB --user-data-dir=chrome/tagui_user_profile --disable-oor-cors --enable-auto-reload --num-raster-threads=2 --enable-zero-copy --enable-gpu-memory-buffer-compositor-resources --enable-main-frame-before-activation --service-request-channel-token=6669127898619523836 --renderer-client-id=30 --no-v8-untrusted-code-mitigations --seatbelt-client=88
14362 s000  S      0:00.79 /Applications/Google Chrome.app/Contents/Frameworks/Google Chrome Framework.framework/Versions/79.0.3945.88/Helpers/Google Chrome Helper (Renderer).app/Contents/MacOS/Google Chrome Helper (Renderer) --type=renderer --remote-debugging-port=9222 --field-trial-handle=1718379636,15956115326844771816,7185594664229212159,131072 --lang=en-GB --user-data-dir=chrome/tagui_user_profile --instant-process --disable-oor-cors --enable-auto-reload --num-raster-threads=2 --enable-zero-copy --enable-gpu-memory-buffer-compositor-resources --enable-main-frame-before-activation --service-request-channel-token=7746585619481135149 --renderer-client-id=31 --no-v8-untrusted-code-mitigations --seatbelt-client=88
```
```
Tebel:src kensoh$ ps x | grep remote-debugging-port=9222 | grep tagui_user_profile | sed -e 's/^[ ]*//'                 
14588 s000  S+     0:06.50 /Applications/Google Chrome.app/Contents/MacOS/Google Chrome --user-data-dir=chrome/tagui_user_profile --remote-debugging-port=9222 about:blank --window-size=1366,842
14747 s000  R+     0:00.65 /Applications/Google Chrome.app/Contents/Frameworks/Google Chrome Framework.framework/Versions/79.0.3945.88/Helpers/Google Chrome Helper (Renderer).app/Contents/MacOS/Google Chrome Helper (Renderer) --type=renderer --remote-debugging-port=9222 --field-trial-handle=1718379636,7252428747212509387,5878820744270324615,131072 --lang=en-GB --user-data-dir=chrome/tagui_user_profile --disable-oor-cors --enable-auto-reload --num-raster-threads=2 --enable-zero-copy --enable-gpu-memory-buffer-compositor-resources --enable-main-frame-before-activation --service-request-channel-token=14471844672944768107 --renderer-client-id=23 --no-v8-untrusted-code-mitigations --seatbelt-client=119
14759 s000  S+     0:00.12 /Applications/Google Chrome.app/Contents/Frameworks/Google Chrome Framework.framework/Versions/79.0.3945.88/Helpers/Google Chrome Helper (Renderer).app/Contents/MacOS/Google Chrome Helper (Renderer) --type=renderer --remote-debugging-port=9222 --field-trial-handle=1718379636,7252428747212509387,5878820744270324615,131072 --lang=en-GB --user-data-dir=chrome/tagui_user_profile --disable-oor-cors --enable-auto-reload --num-raster-threads=2 --enable-zero-copy --enable-gpu-memory-buffer-compositor-resources --enable-main-frame-before-activation --service-request-channel-token=15579972353932130591 --renderer-client-id=24 --no-v8-untrusted-code-mitigations --seatbelt-client=157
```

**Comment by kensoh** [comment]
Fix deployed in TagUI v5.11 packaged release - https://github.com/kelaberetiv/TagUI#set-up

Solution is add an additional `grep window-size` to the process filter in `tagui/src/tagui` file. Tested to be working on macOS and Linux. For Windows, tested quite exhaustively with @siowyisheng that the latest Chrome version is not impacted by this, so no change was made.

**Comment by kensoh** [comment]
Users with with an existing installation, you may also download the cutting edge version master.zip to overwrite. Instead of downloading above packaged zip which is 150+ MB.

More details here - https://github.com/kelaberetiv/TagUI#set-up
